ci-operator/step-registry/baremetalds/assisted/images/publish/baremetalds-assisted-images-publish-commands.sh

```diff
@@ -8,17 +8,17 @@ cd "$HOME" || exit 1
 if [[ "$JOB_TYPE" == "periodic" ]]; then
     echo "INFO JOB_TYPE is: $JOB_TYPE populating repo metadata from JOB_SPEC"
 
-    REPO_OWNER=$(echo "${JOB_SPEC}" | jq '.extra_refs[].org')
-    REPO_NAME=$(echo "${JOB_SPEC}" | jq '.extra_refs[].repo')
-    PULL_BASE_REF=$(echo "${JOB_SPEC}" | jq '.extra_refs[].base_ref')
+    REPO_OWNER=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].org')
+    REPO_NAME=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].repo')
+    PULL_BASE_REF=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].base_ref')
 fi
 
-# Uncomment the below to test JQ commands
-# REPO_OWNER_TEST=$(echo "${JOB_SPEC}" | jq '.extra_refs[].org')
-# REPO_NAME_TEST=$(echo "${JOB_SPEC}" | jq '.extra_refs[].repo')
-# PULL_BASE_REF_TEST=$(echo "${JOB_SPEC}" | jq '.extra_refs[].base_ref')
+#   Uncomment the below to test JQ commands
+# REPO_OWNER_TEST=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].org')
+# REPO_NAME_TEST=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].repo')
+# PULL_BASE_REF_TEST=$(echo ${JOB_SPEC} | jq -r '.extra_refs[].base_ref')
 # echo "JQ TEST: ${REPO_OWNER_TEST} ${REPO_NAME_TEST} ${PULL_BASE_REF_TEST}"
-# echo "${JOB_SPEC}"
+# echo ${JOB_SPEC}
```

Comment:
Here we should also remove the quotes?

Reply:
prob not, but it shouldn't break. I will approve this PR and we can add the quotes in a follow-up one

Reply:
It works with and without them. Originally I thought these were the culprit, but `jq` was the culprit instead.

Reply:
yeah, my only vote for adding them here is because shellcheck would fail because of the missing quotes :P